go/tools/git/git.go

```diff
@@ -135,7 +135,7 @@ func GetLatestVitessReleaseCommitHash(repoDir string) ([]*Release, error) {
 	}
 	var latestReleases []*Release
 	for _, release := range allReleases {
-		if release.Number[0] > 6 {
+		if release.Number[0] >= 12 {
```

Comment:
We should at some point automate this to last 3 releases or something like that, instead of a hard-coded number. But this is fine for now too.

Reply:
I agree on that.

Reply:
agreed